src/Operation/DropCollection.php

```diff
@@ -40,7 +41,16 @@ public function __construct($databaseName, $collectionName)
      */
     public function execute(Server $server)
     {
-        $cursor = $server->executeCommand($this->databaseName, new Command(array('drop' => $this->collectionName)));
+        try {
+            $cursor = $server->executeCommand($this->databaseName, new Command(array('drop' => $this->collectionName)));
+        } catch (DriverRuntimeException $e) {
+            if ($e->getMessage() === 'ns not found') {
+                $result = (object) ['ok' => 0, 'errmsg' => 'ns not found'];
```

Comment:
Whats the $result used for here?
Is the throw below supposed to be in an else {} ?